Gemfile

```diff
@@ -32,9 +32,6 @@ gem 'delayed_job_active_record'
 # web interface for delayed job
 gem 'delayed_job_web', '>= 1.2.0'
 
-# collection of handy tools
-gem 'handy'
-
```

Comment:
We have some useful tasks here - https://github.com/bigbinary/handy/blob/master/lib/handy/additional_test_directories.rb